MIGRATIONS.md

```diff
@@ -0,0 +1,5 @@
+## March 27 2018 (related to issue #301 and the PRs associated with it)
+- You will have to either download a new database dump or run `npm run db:migrate` (notice that these migrations are temporary and will probably be deleted or moved since they aren't proper migrations since the first one doesn't start from an empty database, so they are only useful for this specific purpose of migrations from current schema to next one)
```

Comment:
Adding a full stop somewhere here would be useful :)

Reply:
Will do!